tox.ini

```diff
@@ -47,22 +47,32 @@ deps =
     numpy120: numpy==1.20.*
     numpy121: numpy==1.21.*
 
-    astropylts: astropy==4.0.*
+    astropy50: astropy==5.0.*
+    astropylts: astropy==5.0.*
+    astropyrc: astropy==5.1rc1
```

Comment:
@bsipocz has set up a way to automatically pick up astropy prerelease over at astropy/astroquery#2404 if you are interested. FYI.

Reply:
oh cool. i'll take a look.

Reply:
yeah! i like that approach and implemented it as one of the cron tests.